homeassistant/components/lcn/config_flow.py

```diff
@@ -61,43 +81,71 @@ async def validate_connection(host_name: str, data: ConfigType) -> ConfigType:
         host, port, username, password, settings=settings
     )
 
-    await connection.async_connect(timeout=5)
+    try:
+        await connection.async_connect(timeout=5)
+        _LOGGER.debug("LCN connection validated")
+    except pypck.connection.PchkAuthenticationError:
+        _LOGGER.warning('Authentication on PCHK "%s" failed', host_name)
+        error = "authentication_error"
+    except pypck.connection.PchkLicenseError:
+        _LOGGER.warning(
+            'Maximum number of connections on PCHK "%s" was '
+            "reached. An additional license key is required",
+            host_name,
+        )
+        error = "license_error"
+    except (TimeoutError, ConnectionRefusedError):
+        _LOGGER.warning('Connection to PCHK "%s" failed', host_name)
+        error = "connection_refused"
 
-    _LOGGER.debug("LCN connection validated")
     await connection.async_close()
-    return data
+    return error
 
 
-class LcnFlowHandler(ConfigFlow, domain=DOMAIN):
+class LcnFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     """Handle a LCN config flow."""
 
     VERSION = 1
 
-    async def async_step_import(self, data: ConfigType) -> ConfigFlowResult:
+    async def async_step_import(
+        self, data: ConfigType
+    ) -> config_entries.ConfigFlowResult:
         """Import existing configuration from LCN."""
-        host_name = data[CONF_HOST]
         # validate the imported connection parameters
-        try:
-            await validate_connection(host_name, data)
-        except pypck.connection.PchkAuthenticationError:
-            _LOGGER.warning('Authentication on PCHK "%s" failed', host_name)
-            return self.async_abort(reason="authentication_error")
-        except pypck.connection.PchkLicenseError:
-            _LOGGER.warning(
-                (
-                    'Maximum number of connections on PCHK "%s" was '
-                    "reached. An additional license key is required"
-                ),
-                host_name,
+        if error := await validate_connection(data):
+            async_create_issue(
+                self.hass,
+                DOMAIN,
+                f"{error}_{DOMAIN}",
```

Comment:
There's no need to suffix with DOMAIN here.
```suggestion
                error,
```